### DISCUSSION TITLE
How to save cookie set in response to browser?

### DISCUSSION THREAD

**Discussion by timmoti** [sindresorhus/ky#503] [Q&A] [answered]
I have a `Set-Cookie` header in my response object with value `token=sessiontoken1234; path=/; expires=Thu, 04 May 2023 02:00:29 GMT; samesite=lax; httponly`, generated by a koa backend.

Using the browser fetch api, I am able to easily set the cookie in browser with a simple fetch call:
`await fetch('http://localhost:4000/set-session', { method: 'GET', cache: 'no-store', credentials: 'include' })`

However I am unable to do so via ky:
`ky.get('http://localhost:4000/set-session')`

I thought I might be able to perform a separate set cookie step in an `afterReponse` hook but accessing `response.headers` results in an empty object: `Headers: {}`. This seems to be the case with the fetch api as well so I think that is the expected behaviour

How do I get ky to set the cookie in the same way as is possible through the fetch api?

Some details:
ky version: **0.33.3**
chrome version: **112.0.5615.121 (Official Build) (arm64)**

**Comment by sindresorhus** [comment, marked as answer]
Did you try `await ky.get('http://localhost:4000/set-session', { cache: 'no-store', credentials: 'include' })`?

**Reply by timmoti** [reply]
Ah yes. That's the reason why. Didn't realise we could use the `RequestInit` options object from the fetch API in the same way for ky. I looked through the types file and it looked sufficiently different.

Thanks for your help!
